emacs-ide.md

```diff
@@ -236,7 +236,7 @@ Compile a **region** by selecting the first 2 forms in test-all and
 running `M-x slime-compile-region`.
 
 Compile a **defun** by putting the cursor inside the "test-format"
-defun and pressing `C-c C-e`.
+defun and pressing `C-c C-c`.
 
 Compile the **sexp** before the point by putting the cursor after the
 closing paren of `(test-format)` and pressing `C-c C-e`.
```

Comment:
Doesn't this command run eval instead of compile?

Reply:
C-c C-c runs compile. C-c C-e runs evaluate.

Reply:
> Compile […] pressing `C-c C-e`.
> ^^^                                      ^^

then it should say "Evaluate"?

Reply:
Also, I think evaluation commands are wrong there?

C-c C-e, prompts for evaluation.

But the evaluation described there I think corresponds to C-x C-e (slime-eval-last-expression).

https://common-lisp.net/project/slime/doc/html/Evaluation.html

Reply:
I hope I'm not getting confused here ...

Reply:
> > Compile […] pressing `C-c C-e`.
> > ^^^                                      ^^
> 
> then it should say "Evaluate"?

I thought compilation was being described there: "Compile a **defun** by putting the cursor inside the "test-format""

So I replaced eval command by compile command.

Reply:
I'm very sorry, my quote was misleading. Your change is right, I was wondering about the next sentence!

Not a big deal, you can leave your change and end the discussion, it's fine!

Reply:
Ok. I think we may want to revise the evaluation commands after. I think they should be C-x C-e instead of C-c C-e most of the time.